src/components/DonateOverlay.tsx

```
@@ -0,0 +1,45 @@
+import { useContext } from "react";
+import { GeneralDonationContext } from "../context/GeneralDonationContext";
+import { Backdrop, Theme } from "@mui/material";
+import DialogCloseButton from "../styles/DialogCloseButton";
+
+const FUNDRAISEUP_GENERAL_CAMPAIGN_CODE =
+  import.meta.env.VITE_FUNDRAISEUP_CAMPAIGN_CODE || "#XWQCRFLJ";
```

It is bad practice to include defaults of DEV data. Something could happen during deployment which would lead the application to expose sensitive information.  It is better to let the application fail on load.